src/mongocrypt-key.c

```diff
@@ -289,20 +302,13 @@ _mongocrypt_key_new ()
    _mongocrypt_key_doc_t *key_doc;
 
    key_doc = (_mongocrypt_key_doc_t *) bson_malloc0 (sizeof *key_doc);
+   BSON_ASSERT (key_doc);
```

Comment:
Redundant assertion: `bson_malloc0` is guaranteed to return a not-null pointer.

Reply:
Removed.  Change coming in next commit.